README.md

```diff
@@ -429,6 +433,30 @@ Response (fault):
 }
 ```
 
+#### GetHealthcheck
+Request:
+```json
+{
+  "type": "jsonwsp/request",
+  "version": "1.0",
+  "servicename": "ogmios",
+  "methodname": "GetHealthcheck",
+  // optional
```

Comment:
Since JSON doesn't have comments this renders a big strangely

Reply:
TIL we can change `json` to `jsonc` and comments work fine 
```jsonc
// comment
"foo": "bar"
```